packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -11,15 +11,27 @@ class CupertinoColors {
 
   /// iOS 10's default blue color. Used to indicate active elements such as
   /// buttons, selected tabs and your own chat bubbles.
+  ///
+  /// This is SystemBlue in the iOS palette.
   static const Color activeBlue = Color(0xFF007AFF);
 
   /// iOS 10's default green color. Used to indicate active accents such as
   /// the switch in its on state and some accent buttons such as the call button
   /// and Apple Map's 'Go' button.
+  ///
+  /// This is SystemGreen in the iOS palette.
   static const Color activeGreen = Color(0xFF4CD964);
 
+  /// iOS 12's default dark mode color. Used in place of the [activeBlue] color
```

Comment:
You may want to avoid the term "dark mode" as per our conversation today.

Reply:
I forgot what we said :S. Can you remind me?